tests/test-war-simple/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM ${docker.image.name}:${docker.tag.long}
```

Comment:
I'd recommend combining these two fields into a `test.docker.image` property, which you can use here in the dockerfile. That will make it easier to run these tests on any image that we might want to substitute in here.